File: ArraylistwithListview/app/src/main/java/com/example/arraylistwithlistview/MainActivity.java
package com.example.arraylistwithlistview;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ListView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    ListView list;
    ArrayList<String> food = new ArrayList<>();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        food();
    }

    private void food()
    {
        list = findViewById(R.id.list);
        food.add("Pizza");
        food.add("Burger");
        food.add("HotDog");
        food.add("Momos");
        food.add("Panipuri");
        food.add("Sandwich");
        food.add("Manchuriyam");
        food.add("Dhosa");
        food.add("Noddles");
        food.add("Gathiya");

        FoodAdapter adapter = new FoodAdapter(food,MainActivity.this);
        list.setAdapter(adapter);

        list.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
            @Override
            public void onItemSelected(AdapterView<?> adapterView, View view, int i, long l) {

            }

            @Override
            public void onNothingSelected(AdapterView<?> adapterView) {

            }
        });
    }
}

File: ImageSlider/app/src/main/java/com/example/imageslider/MainActivity.kt
package com.example.imageslider

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.TextView
import androidx.viewpager.widget.ViewPager
import java.util.function.IntFunction

class MainActivity : AppCompatActivity() {
   lateinit var txtCount : TextView
lateinit var imageViewMain : ViewPager
var images = intArrayOf(R.drawable.image1,
        R.drawable.image2,R.drawable.image3,R.drawable.image4,R.drawable.image5)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initview()
    }

    private fun initview() {
        imageViewMain = findViewById(R.id.imageViewMain)
        txtCount = findViewById(R.id.txtCount)

        imageViewMain.addOnPageChangeListener(object : ViewPager.OnPageChangeListener{
            override fun onPageScrolled(
                position : Int,
                positionOffset : Float,
                positionOffsetPixels : Int
            ){
                Log.e("TAG", "onPageScrolled: " )
            }

            override fun onPageSelected(position: Int) {
                txtCount.text="${position+1}/5"
            }

            override fun onPageScrollStateChanged(state: Int) {
            }
        })

        var ImageAdapter = ImageAdapter(images)
        imageViewMain.adapter = ImageAdapter
    }
}